archetypes/states.md

```diff
@@ -0,0 +1,7 @@
++++
+title = ""
+slug = ""
+state_abbreviation = ""
+registration_type = ""
+external_link = ""
++++
```

Comment:
This archetype will be used to generate new `states` or `territories` using `hugo new states/<NAME>` as needed. This generates a default template that will be modified to have it included in the drop down.